REST/tests/SyncListenerTest.cc

```diff
@@ -320,6 +320,10 @@ TEST_CASE_METHOD(C4SyncListenerTest, "P2P Server Addresses", "[Listener]") {
     }
     
     for(const auto& interface : litecore::net::Interface::all()) {
+        if(!interface.isRoutable()) {
```

Comment:
This change is unrelated to the base topic?

Reply:
Yes, it was bugging me though because the test fails every time on my machine without it...

Reply:
I took it out though, will put it back later